el/src/main/java/com/sun/ts/tests/el/api/jakarta_el/arrayelresolver/ELClientJspTest.java

```diff
@@ -0,0 +1,444 @@
+
+    WebArchive jsp_vehicle_web = ShrinkWrap.create(WebArchive.class, "arrayelresolver_jsp_vehicle_web.war");
+  
+    jsp_vehicle_web.addClasses(
+        com.sun.ts.lib.harness.EETest.Fault.class,
+        com.sun.ts.lib.harness.EETest.SetupException.class,
+        com.sun.ts.lib.harness.EETest.class,
+        com.sun.ts.lib.harness.ServiceEETest.class,
+        com.sun.ts.tests.common.el.api.resolver.BarELResolver.class,
+        com.sun.ts.tests.common.el.api.resolver.ResolverTest.class,
+        com.sun.ts.tests.common.vehicle.VehicleClient.class,
+        com.sun.ts.tests.common.vehicle.VehicleRunnable.class,
+        com.sun.ts.tests.common.vehicle.VehicleRunnerFactory.class,
+        com.sun.ts.tests.el.common.elcontext.BareBonesELContext.class,
+        com.sun.ts.tests.el.common.util.ELTestUtil.class,
+        com.sun.ts.tests.el.api.jakarta_el.arrayelresolver.ELClientJspTest.class
+    );
+
+    InputStream inStream = ELClientJspTest.class.getResourceAsStream("/vehicle/jsp/jsp_vehicle_web.xml");
+
+    // Replace the el_jsp_vehicle in jsp_vehicle_web.xml with the jsp_vehicle name for archive
+    String webXml = editWebXmlString(inStream, VEHICLE_ARCHIVE);
+    jsp_vehicle_web.setWebXML(new StringAsset(webXml));
+
+    // Web content
+    URL warResURL = ELClientJspTest.class.getResource("/vehicle/jsp/contentRoot/client.html");
+    jsp_vehicle_web.addAsWebResource(warResURL, "/client.html");
+    warResURL = ELClientJspTest.class.getResource("/vehicle/jsp/contentRoot/jsp_vehicle.jsp");
+    jsp_vehicle_web.addAsWebResource(warResURL, "/jsp_vehicle.jsp");
+
+    EnterpriseArchive jsp_vehicle_ear = ShrinkWrap.create(EnterpriseArchive.class, "arrayelresolver_jsp_vehicle.ear");
+    jsp_vehicle_ear.addAsModule(jsp_vehicle_web);
+    return jsp_vehicle_ear;
+
+  }
+
+  public static String inputStreamToString(InputStream inStream) throws IOException {
+    try (BufferedReader bufReader = new BufferedReader(new InputStreamReader(inStream, StandardCharsets.UTF_8))) {
+      return bufReader.lines().collect(Collectors.joining(System.lineSeparator()));
+    }
+  }
+
+  public static String editWebXmlString(InputStream inStream, String servlet_vehicle) throws IOException{
+    return inputStreamToString(inStream).replaceAll("el_jsp_vehicle", servlet_vehicle);
+  }
+  
+  @BeforeEach
+  void logStartTest(TestInfo testInfo) {
+      logger.log(Logger.Level.INFO, "STARTING TEST : " + testInfo.getDisplayName());
+  }
+
+  @AfterEach
+  void logFinishTest(TestInfo testInfo) {
+      logger.log(Logger.Level.INFO, "FINISHED TEST : " + testInfo.getDisplayName());
+  }
+
+  private Properties testProps;
+
+  private static final String[] names = { "doug", "nick", "roger", "ryan",
+      "ed" };
+
+  public static void main(String[] args) {
+    ELClientJspTest theTests = new ELClientJspTest();
+    Status s = theTests.run(args, System.out, System.err);
+    s.exit();
+  }
+
+  public void setup(String[] args, Properties p) throws Exception {
```

Comment:
@starksm64 I see that the setup method is not invoked to set the properties for all the ClientTest classes. Do you think I am missing anything here?
cc @gurunrao

Reply:
Please verify if properties for EL tests are present @ https://github.com/eclipse-ee4j/jakartaee-tck-tools/blob/bf59506a644b4194ce4528e6defe66465b46a8a3/arquillian/common/src/main/java/tck/arquillian/protocol/common/TsTestPropsBuilder.java#L21 
Javatest protocol will only read properties present at https://github.com/eclipse-ee4j/jakartaee-tck-tools/blob/bf59506a644b4194ce4528e6defe66465b46a8a3/arquillian/common/src/main/java/tck/arquillian/protocol/common/TsTestPropsBuilder.java#L21

Reply:
Okey. I see "variable.mapper" is missing which is required for EL tests. Can it be added.

Reply:
@starksm64 - Please release new jakartaeetck-tools with above property.

Reply:
https://github.com/eclipse-ee4j/jakartaee-tck-tools/pull/147